apps/remote_control/lib/lexical/remote_control/mix.ex

```diff
@@ -47,4 +46,14 @@ defmodule Lexical.RemoteControl.Mix do
       end
     end)
   end
+
+  if Mix.env() == :test do
+    defp unique_project_name(%Project{} = project) do
+      Project.atom_name(project)
+    end
+  else
+    defp unique_project_name(%Project{} = project) do
+      :"#{Project.name(project)}.#{System.unique_integer()}"
```

Comment:
This is for solving the multiple editors opening the same project problem, right?

Reply:
These changes will cause some problems. Project node can't automatically be closed.

Reply:
Please try to address one goal with each PR as much as possible

Reply:
Nope, I was getting a build error, and having a unique name fixed it, but it broke formatting.

Reply:
I figured out the actual problem; it was a naming conflict with the `lexical` project in the `projects` directory and the main app. I renamed the one in the `projects` directory to `lexical_shared`